python/sglang/srt/server_args.py

```diff
@@ -183,7 +183,8 @@ class ServerArgs:
     enable_flashmla: bool = False
     flashinfer_mla_disable_ragged: bool = False
     warmups: Optional[str] = None
-    n_share_experts_fusion: int = 0
+    n_share_experts_fusion: Optional[int] = None
+    routed_scaling_factor: Optional[float] = None
```

Comment:
nit: it seems that the scaling factor will be picked by the model config, so maybe we do not need to pass in as an arg